### ISSUE TITLE
Undefined data.title.$t in new SpreadSheets() call.

### ISSUE THREAD

**Issue by martensms** [samcday/node-google-spreadsheets#19]
I'm getting an error whilst using the API, it seems as the returned data doesn't have data.title set and this is undefined.

Note: The key of the spreadsheet is correct, so it works when I replace /d/<key>/edit/ in the google spreadsheet URL. The spreadsheet is available via link sharing access.

What else do I need to do in order to work with this library?
I tried out username/password as well, but still no success.

My Code:
```javascript

var config       = require('./config');
var Spreadsheets = require('google-spreadsheets');

var sheet = new Spreadsheets({
	key: config.google.sheet
}, function(err, spreadsheet) {

	console.log(spreadsheet.worksheets[0]);

});
```

Stacktrace:

```bash
    /home/christoph/Software/_stuff_/backend/node_modules/google-spreadsheets/lib/spreadsheets.js:165
    	this.title = data.title.$t;
	                 ^
    TypeError: Cannot read property 'title' of undefined
        at new Spreadsheet (/home/christoph/Software/_stuff_/backend/node_modules/google-spreadsheets/lib/spreadsheets.js:165:19)
        at /home/christoph/Software/_stuff_/backend/node_modules/google-spreadsheets/lib/spreadsheets.js:73:12
        at Request._callback (/home/christoph/Software/_stuff_/backend/node_modules/google-spreadsheets/lib/spreadsheets.js:56:3)
        at Request.self.callback (/home/christoph/Software/_stuff_/backend/node_modules/google-spreadsheets/node_modules/request/request.js:372:22)
        at Request.emit (events.js:110:17)
        at Request.<anonymous> (/home/christoph/Software/_stuff_/backend/node_modules/google-spreadsheets/node_modules/request/request.js:1317:14)
        at Request.emit (events.js:129:20)
        at IncomingMessage.<anonymous> (/home/christoph/Software/_stuff_/backend/node_modules/google-spreadsheets/node_modules/request/request.js:1265:12)
        at IncomingMessage.emit (events.js:129:20)
        at _stream_readable.js:907:16

```

**Comment by samcday** [comment]
I think this might be related to #20 - it seems there's a new Google Spreadsheets format that newly created spreadsheets are using. I haven't had a chance to look into this though. If anyone has investigated further and can submit a PR that would be awesome.

**Comment by fitch** [comment]
Anybody looked into this?

**Comment by samcday** [comment]
@fitch this should be resolved in version `0.4.0` of this library. More info in #20
